composer.json

```
@@ -3,32 +3,30 @@
   "description": "A variety of standard config, scripts and packages to support GovCMS scaffold.",
   "license": "GPL-2.0-or-later",
   "require": {
-    "drush/drush": "^12",
+    "drush/drush": "12.4.4",
```

Latest version being installed in 12.5.2, according to latest build failure
```suggestion
    "drush/drush": "12.5.2",
```